distro.py

```diff
@@ -37,8 +37,8 @@
 import argparse
 import subprocess
 
-
 _UNIXCONFDIR = os.environ.get('UNIXCONFDIR', '/etc')
+_UNIXLIBSDIR = os.environ.get('UNIXLIBSDIR', '/usr/lib')
```

Comment:
I find the `S` in `UNIXLIBSDIR` weird. I would vote for `UNIXLIBDIR` without `S` or `_UNIXUSRLIBDIR`. Just an idea.

Reply:
Second suggestion applied.